platform/tests/Platform.ApiTests/Steps/EstablishmentSchoolsSteps.cs

```diff
@@ -117,91 +124,114 @@ private async Task ThenTheSchoolResultShouldBeOk()
         var content = await response.Content.ReadAsByteArrayAsync();
         var result = content.FromJson<School>();
 
-        result.SchoolName.Should().Be("Burscough Bridge St John's Church of England Primary School");
-        result.URN.Should().Be("119376");
+        table.CompareToInstance(result);
     }
 
-    [Given("a invalid school request")]
-    private void GivenAInvalidSchoolRequest()
+    [Then("the school result should be not found")]
+    private void ThenTheSchoolResultShouldBeNotFound()
     {
-        _api.CreateRequest(NotFoundRequestKey, new HttpRequestMessage
-        {
-            RequestUri = new Uri("/api/school/invalid-urn", UriKind.Relative),
-            Method = HttpMethod.Get
-        });
+        var result = _api[RequestKey].Response;
+
+        result.Should().NotBeNull();
+        result.StatusCode.Should().Be(HttpStatusCode.NotFound);
     }
 
-    [Then("the school result should be not found")]
-    private void ThenTheSchoolResultShouldBeNotFound()
+    [Then("the school suggest result should be ok and have the following values:")]
+    private async Task ThenTheSchoolsSuggestResultShouldShouldHaveValues(Table table)
     {
-        var response = _api[NotFoundRequestKey].Response;
+        var response = _api[SuggestRequestKey].Response;
 
         response.Should().NotBeNull();
-        response.StatusCode.Should().Be(HttpStatusCode.NotFound);
+        response.StatusCode.Should().Be(HttpStatusCode.OK);
+
+        var content = await response.Content.ReadAsByteArrayAsync();
+        var results = content.FromJson<SuggestResponse<School>>().Results;
+        var result = results.FirstOrDefault();
+        result.Should().NotBeNull();
+
+        var actual = new
+        {
+            result?.Text,
+            result?.Document?.SchoolName,
+            result?.Document?.URN
+        };
+
+        table.CompareToInstance(actual);
```

Comment:
Is it possible to use `result` instead of `actual` here and below, or are there too many fields that are out of our control in the response?